github-fetcher/json-orm_test.go

```diff
@@ -0,0 +1,292 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/google/go-github/github"
+)
+
+type issueTest struct {
+	gIssue        *github.Issue
+	expectedIssue *Issue
+}
+
+func makeIssue(number int,
```

Comment:
Actually, new can only create zero values.

http://stackoverflow.com/questions/30716354/how-do-i-do-a-literal-int64-in-go

The simplest way seem to be: `&[]string{"String"}[0]`. Very elegant =)